lib/plugins/package/lib/zipService.js

```diff
@@ -2,12 +2,15 @@
 
 /* eslint-disable no-use-before-define */
 /* eslint-disable no-param-reassign */
+/* eslint-disable consistent-return */
```

Comment:
Addition: Imo it's generally bad to disable linter check per file. It would be better to put an `elslint-disable-line consistent-return` at the specific location and comment why this rule is needed to be overridden there.